lib/msal-browser/src/interaction_client/NativeInteractionClient.ts

```diff
@@ -88,7 +104,22 @@ export class NativeInteractionClient extends BaseInteractionClient {
                     success: false,
                     errorCode: error.errorCode,
                     subErrorCode: error.subError,
-                    isNativeBroker: true
+                    isNativeBroker: true,
+                    extensionId: this.nativeMessageHandler.getExtensionId(),
+                    extensionVersion: this.nativeMessageHandler.getExtensionVersion(),
+                    mats_broker_version: mats ? mats.broker_version : undefined,
+                    mats_account_join_on_start: mats ? mats.account_join_on_start : undefined,
+                    mats_account_join_on_end: mats ? mats.account_join_on_end : undefined,
+                    mats_device_join: mats ? mats.device_join : undefined,
+                    mats_prompt_behavior: mats ? mats.prompt_behavior : undefined,
+                    mats_api_error_code: mats ? mats.api_error_code : undefined,
+                    mats_ui_visible: mats ? mats.ui_visible : undefined,
+                    mats_silent_code: mats ? mats.silent_code : undefined,
+                    mats_silent_bi_sub_code: mats ? mats.silent_bi_sub_code : undefined,
+                    mats_silent_message: mats ? mats.silent_message : undefined,
+                    mats_silent_status: mats ? mats.silent_status : undefined,
+                    mats_http_status: mats ? mats.http_status : undefined,
+                    mats_http_event_count: mats ? mats.http_event_count : undefined
```

Comment:
This looks like a submeasurement, and the only property pulled from submeasurements and included in the top level measurement is `durationMs`. Can this logic be moved up to where acquireTokenSilent, etc is called?